int32 SOC
int32 SOH
float32 current
float32 voltage
int32 DCDC 
